### ISSUE TITLE
Problem Creating Array Tags

### ISSUE THREAD

**Issue by Ryushi5** [tnunnink/L5Sharp#17]
I'm running into an issue with generating an array tag. For this example, I'm making a BOOL[100] tag.

For the Tag creation, I am setting the Value to the following:
`Value = ArrayType.New<BOOL>(100)`

It could be that there is a better way to do this, but it seems to work except for one thing. The resulting L5X tag that's exported shows its DataType as:
`... DataType="BOOL[100]" Radix="Decimal" Dimensions="100">`

This does not import into Studio 5000, but works just fine if the DataType is "BOOL" with the Dimensions set at "100".

**Comment by tnunnink** [comment]
Thanks for reporting this. I forget why I added the index to the data type name for array types. I think I wanted a way to distinguish non-array types from array types, but it has been a while, so I'm not sure. I need to go back and review this and some other things and do some more testing when I find time.

In the interim I have fixed the issue and published a new version [0.19.7](https://www.nuget.org/packages/L5Sharp/0.19.7) which should write the correct name to the DataType attribute for array tags.
